config/settings/common.py

```diff
@@ -70,6 +71,11 @@
 
 # Email
 EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = os.getenv("EMAIL_HOST", "localhost")
+EMAIL_PORT = os.getenv("EMAIL_PORT") or 1025
+EMAIL_HOST_USER = os.getenv("EMAIL_HOST_USER")
+EMAIL_HOST_PASSWORD = os.getenv("EMAIL_HOST_PASSWORD")
+DEFAULT_FROM_EMAIL = "UOB Theatre <no-reply@uobtheatre.com>"
```

Comment:
Can this be moved into `local.py`? (Since we have a different setup for production)

Reply:
the default backend is defined in each (i.e. local and prod) but the host, port and user/pass stuff is should be defined in common so we can use SMTP instead of SES for example in production. I don't think we want to lock our production into just SES